### ISSUE TITLE
[RFE] add way to list default options

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1390]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1386
- Created at 2012-06-21 15:32:40 by [simo](https://pagure.io/user/simo)
- Closed at 2020-03-24 14:21:10 as wontfix
- Assigned to nobody
---

It would be nice to be able to run sssd -b and that would sim ply print all the configure time options that have been set including defaults for various configuration options.

(This means manpages can point to this command instead of mentioning defaults that may be different on different builds)

### Comments

---

**Comment from [simo](https://pagure.io/user/simo) at [2012-06-29 15:26:25](https://pagure.io/SSSD/sssd/issue/1386#242934)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD Deferred
rhbz:  => 0

---

**Comment from [simo](https://pagure.io/user/simo) at [2013-12-02 21:19:54](https://pagure.io/SSSD/sssd/issue/1386#242935)**

Fields changed

type: defect => enhancement

---

**Comment from [simo](https://pagure.io/user/simo) at [2014-03-07 15:45:20](https://pagure.io/SSSD/sssd/issue/1386#242936)**

Fields changed

changelog:  => 
design:  => 
design_review:  => 0
fedora_test_page:  => 
keywords:  => Status
review:  => 0
selected:  =>

---

**Comment from [simo](https://pagure.io/user/simo) at [2014-03-07 15:52:45](https://pagure.io/SSSD/sssd/issue/1386#242937)**

Fields changed

summary: RFE: add way to list default options => [RFE] add way to list default options

---

**Comment from [simo](https://pagure.io/user/simo) at [2016-11-17 11:33:24](https://pagure.io/SSSD/sssd/issue/1386#242938)**

Fields changed

blockedby:  => #3157
mark:  => 0
sensitive:  => 0

---

**Comment from [simo](https://pagure.io/user/simo) at [2017-02-24 15:07:44](https://pagure.io/SSSD/sssd/issue/1386#242939)**

**Metadata Update from @simo**:
- Issue marked as blocked by: #1828
- Issue marked as depending on: #3157
- Issue set to the milestone: SSSD Patches welcome

---

**Comment from [simo](https://pagure.io/user/simo) at [2020-03-24 14:21:09](https://pagure.io/SSSD/sssd/issue/1386#636959)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [simo](https://pagure.io/user/simo) at [2020-03-24 14:21:11](https://pagure.io/SSSD/sssd/issue/1386#636960)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
